pkg/kanctl/repositoryserver.go

```diff
@@ -0,0 +1,233 @@
+// Copyright 2023 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kanctl
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
+)
+
+const (
+	tlsSecretFlag                       = "tlsSecret"
+	repoServerUserFlag                  = "repositoryServerUser"
+	repoServerUserAccessSecretFlag      = "repositoryServerUserAccessSecret"
+	repoServerAdminUserAccessSecretFlag = "repositoryServerAdminUserAccessSecret"
+	repoPasswordSecretFlag              = "repositoryPasswordSecret"
+	repoUserFlag                        = "repositoryUser"
+	locationCredsSecretFlag             = "locationCredsSecret"
+	locationSecretFlag                  = "locationSecret"
+	defaultKanisterNamespace            = "kanister"
```

Comment:
Why do we have it as constant? Should we have it as an optional flag instead?

Reply:
which flag are you talking about @ankitjain235 ?

Reply:
Removed the `defaultKanisterNamespace` flag, now we're taking it's value from --namespace flag